data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
I am a bit confused as to what we are arguing about here. 
Right now, the library will be injected in all tabs and on every page load in those tabs, just like `privly.js`. We can't avoid this if we want the content scripts to be present inside an iframe as well. So trimming down the library seems to be the best option(?).